### ISSUE TITLE
Set a default VERSION string

### ISSUE THREAD

**Issue by mweinelt** [esper-hub/esper#7]
What's the point of not compiling any release information into the firmware?

Let's default to something that conveys
- date/time (e.g. `date +%s`)
- commit (`git describe --always`)

```
% echo "$(date +%s)-$(git describe --always)"
1502206675-226a62d
```

**Comment by fooker** [comment]
Like it. But should the commit ID be the one of the ESPer repository or the one of the site project. What happens if the side is not managed by git?

**Comment by mweinelt** [comment]
```
% cd esper-site/esper
% echo "$(date +%s)-e$(git describe --always)-c$(git -C .. describe --always)" 
1503410238-e2767f11-cac0441f
```

Too long?

**Comment by fooker** [comment]
Fixed by 4359d7c.
